src/game_inventory.cpp

```diff
@@ -1835,13 +1835,16 @@ class bionic_sterilize_preset : public inventory_selector_preset
             return loc->has_flag( "NO_STERILE" ) && loc->is_bionic();
         }
 
-        std::string get_denial( const item_location & ) const override {
+        std::string get_denial( const item_location &loc ) const override {
             auto reqs = *requirement_id( "autoclave_item" );
 
             if( !reqs.can_make_with_inventory( p.crafting_inventory(), is_crafting_component ) ) {
                 return pgettext( "volume of water", "2 L" );
             }
 
+            if( loc.get_item()->has_flag( "FILTHY" ) ) {
+                return string_format( _( "CBM is filthy.  Wash it first." ) );
```

Comment:
```suggestion
                return _( "CBM is filthy.  Wash it first." );
```
There is **really** no need for string formatting a single string without any parameters.